.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+
+python:
+  - "2.7"
+
+before_install:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+
+install:
+  - pip install -r requirements.txt
```

Comment:
Since that installs the package with `pip install -e .` that might miss issues related to package-based installation (like missing asset files). See https://github.com/open-craft/xblock-mentoring/blob/00778d0326dee10e9aaea684063b0151b9a818e1/.travis.yml#L12

Besides that, +1 for using Travis :D